src/MessageToast.elm

```diff
@@ -462,6 +507,75 @@ appendToList toastMessage (MessageToast config toasts) =
         |> MessageToast config
 
 
+defaultToastMessage : ToastMessage msg
+defaultToastMessage =
+    { content = Undefined
+    , id = 0
+    , persisted = False
+    , toastType = Danger
+    }
+
+
+dropOldestToast : Bool -> MessageToast msg -> MessageToast msg
+dropOldestToast includePersisted (MessageToast config toasts) =
```

Comment:
Not sure why we need to consider the persisted state here. It seems that it is only used when toasts are not persisted. Put another way... When would drop oldest with persisted toasts be used? Doesn't make sense to me.

Reply:
I was thinking about dropping `getOldestToast` and  `dropOldestToast`.

But decided to keep them and also add `getOldestUnpersistedToast` and `dropOldestUnpersistedToast` to allow users to create their own custom functionality with it.

Maybe users want to create a subscription that works different to ours (maybe they want to do an extra task whenever a message gets dropped). Maybe users want to have their own features to drop toast messages (like a "clear all" button).

Reply:
Ah ok. I guess it doesn't hurt to keep it in, but I do think that a grand majority of users of the library will not need the added flexibility.

Reply:
> I'm still iffy on whether or not we need the added flexibility previously discussed, but I don't feel strongly enough about it to block the PR.

Hm, I'm not entirely sure we should have this either now.

- More code needs to be maintained
- It's code that may not be used at all
- It could lead to users creating their own functionalities. Functionality which we should actually add to the package and thus potentially miss

Removing the functions would definitely be a breaking change. And I'd rather do that now than later.

Maybe we should remove the functions now and see if people are using or need it, and then we can still think about adding the functions back if this seems helpful.

@Tranquility what do you think?

Reply:
I totally agree. An additional benefit of removing all the functions that we don't really need is a simpler API for the user to understand.